### PR TITLE
Feat/login

### PR DESCRIPTION
**Pull request by Vikas2426**
Login Page
image

Signup Page
image

### CODE REVIEW

File: login/login.html [hunk 1]
```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Jackets</title>
```
Comment (Tarandeep-s1ngh) [inline login/login.html:8]: You can give a better title

File: login/style.css [hunk 2]
```diff
@@ -0,0 +1,39 @@
+@import url("../style.css");
+
+.form-container {
+  background-color: var(--white);
+  max-width: 500px;
+  height: 450px;
```
Comment (Tarandeep-s1ngh) [inline login/style.css:6]: better to use relative units like `rems`

File: signup/signup.html [hunk 3]
```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Jackets</title>
```
Comment (Tarandeep-s1ngh) [inline signup/signup.html:8]: You can give a better title over here also

**Review by Tarandeep-s1ngh — APPROVED** [review]

File: login/login.html [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+                    <img src="../assets/Vectorwishlist.svg" alt="">
+                    <span class="badge badge-right">0</span>
+                </div>
+            </a>
+            <a href="../cart.html" class=" badge-container ">
+                <span>
+                    <img src="../assets/Vectorcart.svg" alt="">
+                    <span class="badge badge-right">5</span>
+                </span>
+            </a href="../cart.html">
```
Comment (samflab) [inline login/login.html:33]: this is a closing tag here.

File: login/style.css [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+  font-size: larger;
+}
+.form-container > input[type="text"],
+.form-container > input[type="email"],
+.form-container > input[type="password"] {
+  width: 100%;
+  border: 2px solid var(--primary);
+  padding: 0.5rem 1rem;
+  border-radius: 5px;
+  font-size: large;
```
Comment (samflab) [inline login/style.css:26]: specify the font-size, don't write small, large or medium

**Review by samflab — APPROVED** [review]
everything else looks good just make the above changes.

File: login/login.html (hunk 1 above)
Comment (Vikas2426) [reply login/login.html:8]: done

File: login/style.css (hunk 2 above)
Comment (Vikas2426) [reply login/style.css:6]: i need it to be same size on all screens

File: login/login.html (hunk 4 above)
Comment (Vikas2426) [reply login/login.html:33]: fixed

File: login/style.css (hunk 5 above)
Comment (Vikas2426) [reply login/style.css:26]: fixed

File: signup/signup.html (hunk 3 above)
Comment (Vikas2426) [reply signup/signup.html:8]: fixed

**Pull request merged**